datacalls.go

```diff
@@ -134,3 +147,19 @@ func deletedata(c cbClient, collection_id string, query *Query) error {
 	}
 	return nil
 }
+
+func (d *DevClient) GetColumns(collection_id string) ([]interface{}, error) {
```

Comment:
This is a thing that both Dev's and User's can do, so I think there should be an implementation for UserClient too. @sh3nan1gans @aallsbrook